src/client/dfs/dfs.c

```diff
@@ -562,14 +562,16 @@ insert_entry(daos_handle_t oh, daos_handle_t th, const char *name, size_t len,
 	sgl.sg_iovs	= sg_iovs;
 
 	rc = daos_obj_update(oh, th, flags, &dkey, 1, &iod, &sgl, NULL);
-	if (rc) {
```

Comment:
I still don't understand what you are fixing here.. sorry if im missing something..

Reply:
Right. I checked the original code. There is no need to change insert_entry(). I messed up errno and daos errno yesterday, so I though I need to change it. Now I revert insert_entry() back.